src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -458,18 +454,27 @@ const pickerType = computed(() => ({
 		partialRange: false,
 	},
 	enableTimePicker: !(props.type === 'date' || props.type === 'date-range'),
-	flow: props.type === 'datetime'
-		? ['calendar', 'time'] as ['calendar', 'time']
-		: undefined,
+	...(props.type === 'datetime'
+		? { flow: { steps: ['calendar', 'time'] as ['calendar', 'time'] } }
```

Comment:
```suggestion
		? { flow: { steps: ['calendar', 'time'] as const } }
```

Reply:
does not work as this makes it `readonly` which does not satisfy the types.